crates/jstz_cli/src/config.rs

```diff
@@ -202,4 +196,44 @@ impl Config {
     pub fn accounts(&mut self) -> &mut AccountConfig {
         &mut self.accounts
     }
+
+    pub fn octez_client(&self) -> Result<OctezClient> {
+        let sandbox = self.sandbox()?;
+
+        Ok(OctezClient {
+            octez_client_bin: Some(self.octez_path.join("octez-client")),
+            octez_client_dir: sandbox.octez_client_dir.clone(),
+            endpoint: format!("http://127.0.0.1:{}", sandbox.octez_node_rpc_port),
```

Comment:
I would assume so, but for a different PR.

Reply:
This PR now explicitly reverts this addition of `jstz_node_host` since it incorrectly interacted with the sandbox. I will create a PR that explicitly handles different network types

Reply:
Can you please link the task or PR where it will be handled?

Reply:
https://app.asana.com/0/1205770721173531/1206249753769952/f